addons/mrp/models/mrp_document.py

```diff
@@ -20,7 +20,7 @@ def copy(self, default=None):
         if ir_default:
             ir_fields = list(self.env['ir.attachment']._fields)
             ir_default = {field : default[field] for field in default.keys() if field in ir_fields}
-        new_attach = self.ir_attachment_id.copy(ir_default)
+        new_attach = self.ir_attachment_id.with_context(no_document=True).copy(ir_default)
```

Comment:
The 'no_document' context is used in the enterprise module "documents".
Does using it in a unrelated module break any convention ?